docs/docs/emotion.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Emotion
+---
+
+In this guide, we'll walk through setting up a site with the CSS-in-JS library [Emotion](https://emotion.sh).
+
+Emotion is a performant and flexible CSS-in-JS library. Building on many other CSS-in-JS libraries, it allows you to style apps quickly with string or object styles. It has predictable composition to avoid specificity issues with CSS. With source maps and labels, Emotion has a great developer experience and great performance with heavy caching in production.
+
+First, open a new terminal window and run the following to create a new site:
+
+```shell
+gatsby new emotion-tutorial https://github.com/gatsbyjs/gatsby-starter-hello-world
+```
+
+Second, we'll install the Gatsby plugin for Emotion.
+
+```shell
+npm install --save gatsby-plugin-emotion @emotion/core @emotion/styled
+```
+
+And then add it to your site's `gatsby-config.js`:
+
+```javascript:title=gatsby-config.js
+module.exports = {
+  plugins: [`gatsby-plugin-emotion`],
+}
+```
+
+Then in your terminal run `gatsby develop` to start the Gatsby development server.
```

Comment:
```suggestion
Then in your terminal run `npm start` to start the Gatsby development server.
```

Since we're recommending Node 8 now, we can move off of globals and use the local installed version.